internal/ingress/controller/parser/parser_test.go

```diff
@@ -149,7 +150,7 @@ func TestGlobalPlugin(t *testing.T) {
 							"global": "true",
 						},
 						Annotations: map[string]string{
-							"kubernetes.io/ingress.class": "kong",
+							"kubernetes.io/ingress.class": annotations.DefaultIngressClass,
```

Comment:
Please export `annotations.ingressClassKey` and use it throughout the codebase instead of a literal value of `"kubernetes.io/ingress.class"`.

Reply:
Please create a new issue/card for these--I don't disagree, but I loathe expanding the scope of existing issues/PRs to include additional tasks found in the course of writing them unless it's critical.

Reply:
fixed in commit 36e4d07